shared/notifications/institutional/index.js

```diff
@@ -1,35 +0,0 @@
-export const UI_INSTITUTIONAL_NOTIFICATIONS = {
```

Comment:
Why is this being deleted?

Reply:
Hey @danjm, in a previous [PR](https://github.com/MetaMask/metamask-extension/pull/21146), we removed all MMI notifications because they were not doing anything. We don't currently want to show different notifications than the ones that MM indicates. That's why I'm removing unused code.